### ISSUE TITLE
Update to latest ember and ember-cli

### ISSUE THREAD

**Issue by bcardarella** [ember-cli/ember-cli-todos#32]
Low hanging fruit it someone wants to tackle this. Looks good on us if we have an up to date ember to-do app out there

**Comment by quantuminformation** [comment]
Yeah I would love to see this. I'm thinking of using it as an example for my new version ember[ test-recorder](https://github.com/QuantumInformation/test-recorder)

**Comment by quantuminformation** [comment]
I guess the current version is good enough for now @bcardarella ?

**Comment by broerse** [comment]
I forgot about my never merged PR [#36](https://github.com/ember-cli/ember-cli-todos/pull/36) but perhaps it can be used to go to 2.18.2 ;-)

**Comment by quantuminformation** [comment]
Another thread rises from the dead ;) J

Don't you just love github

**Comment by sandstrom** [comment]
Is this repo ment to mirror https://github.com/tastejs/todomvc/? Perhaps it's easier to just mark this repo as read-only (archived) and add a link in the README directing people to the todomvc repo?

There is a PR there that [updated to 2.10](https://github.com/tastejs/todomvc/pull/1722).
